modules/get-started/pages/whats-new-241.adoc

```diff
@@ -0,0 +1,21 @@
+= What's New in 24.1 Beta
+:description: Summary of new features and updates in the release.
+
+This topic includes new content added in version 24.1 beta. 
+
+== Write caching
+
+xref:develop:config-topics.adoc#configure-write-caching[Write caching] is a relaxed mode of `acks=all` that provides better performance at the expense of durability. It acknowledges a message as soon as it is received and acknowledged on a majority of brokers, without waiting for it to fsync to disk. This provides lower latency while still ensuring that a majority of brokers acknowledge the write. For clusters in development mode, write caching is enabled by default. For clusters in production mode, it is disabled by default.
```

Comment:
We should link to the docs showing where to configure it on topics

Reply:
@mattschumpert We had to merge this, but I'll open a new PR in a minute to add the changes to `rpk security acl` and any other feedback. For this comment: isn't this link at the beginning of the blurb the correct link for configuring write caching?